mne/io/kit/tests/test_kit.py

```diff
@@ -50,6 +51,7 @@ def test_data():
                               hsp=hsp_path, stim=list(range(167, 159, -1)),
                               slope='+', stimthresh=1)
     assert_true('RawKIT' in repr(raw_py))
+    assert_true(raw_mrk.info['kit_system_id'] == 34)
```

Comment:
ok can you then use this dict here in the test?

stick to the dict approach for now